docs/articles/nunit/extending-nunit/Engine-Extensibility.md

```diff
@@ -131,7 +131,7 @@ By use of the `ExtensionPropertyAttribute` the assembly containing this extensio
 
 Of course, this means that the extension author must know a great deal about how each extension point works. That's why we provide a page for each supported extension points with details of how to use it.
 
-### Locating Addins
+### Locating Addins and Extensions
```

Comment:
I wrote the first drafts of this with plans for eventually implementing `addins`, which had a different meaning in the docs. They were more complex and involved cooperating extensions.

In fact, we never impelmented it. Issue nunit/nunit-console#20 is still there but I don't think it's getting any attention.

Anyway, I think we're stuck with the terminology "addins file" but otherwise we should talk about extensions rather than addins.

Reply:
Thanks @CharliePoole -- I'll update this in particular and have created #497 to track the rest.